dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java

```diff
@@ -108,6 +108,10 @@ public void whenCreateRegistrationItShouldPersist() {
         assertThat(addRegistrationResponse1.getStatus()).isEqualTo(HttpStatus.CREATED_201);
         final String registrationId1 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse1);
 
+        /**
+         * work around Jersey client hanging when client1 reused
```

Comment:
I'm going to punt on that until after you figure out if the workaround is truly necessary (like we discussed earlier)